c++.html.markdown

```diff
@@ -867,6 +868,99 @@ v.push_back(Foo());  // New value is copied into the first Foo we inserted
 // explanation of why this works.
 v.swap(vector<Foo>());
 
+
+/////////////////////
+// CONTAINERS
+/////////////////////
+
+//Containers or the Standard Template Library are some predefined templates
+//They  manages the storage space for its elements and provide
+//member functions to access and manipulate them
+
+//Few containers are as follows:-
+
+//Vectors (Dynamic array)
+//Allow us to Define the Array or list of objects at run time
+#include<vector>//will include the header file for vector
+vector< Data_Type > Vector_name; // used to initialize the vector
+cin>>val;
+Vector_name.push_back(val); // will push the value of variable into array
+
+//To iterate through vector, we have 2 choices
+//using normal looping 
+for(int i=0; i<Vector_name.size(); i++)
+//it will iterate through the vector from index '0' till last index
+
+// Using Iterator
+vector<Data_Type>::iterator it; //initialize the iteartor for vector
+for(it=vector_name.begin(); it!=vector_name.end();++it)
+
+//For accessing the element of the vector
+//operator []
+var= vector_name[index]; //will assign value at that index to var
+
+
+//Set
+//Sets are containers that store unique elements following a specific order
+//Very useful container to store unique values in sorted order 
+//without any other functions or code
+
+#include<set>//will include the header file for sets
+set< int > ST; //will initialize the set of int data type
+ST.insert(30);// will insert the value 30 in set ST
+ST.insert(10); // will insert the value 10 in set ST
+ST.insert(20); // will insert the value 20 in set ST
+ST.insert(30);// will insert the value 30 in set ST
+//Now elements of sets are as follows
+ 10 20 30
+ 
+//To erase an element
+ST.erase(20); // will erase element with value 20
+//Set ST: 10 30
+//To iterate through Set we use iterators
+set< int >::iterator it;
+for(it=ST.begin();it<ST.end();it++)
+{
+	cout<<*it<<endl;
```

Comment:
please space operators and brackets (like `a<foo>`) like the rest of the file does